lib/request.js

```diff
@@ -5,6 +5,7 @@
  * Module dependencies.
  */
 
+const URL = require('url').URL;
 const net = require('net');
 const contentType = require('content-type');
 const stringify = require('url').format;
```

Comment:
Thought about destructing here (`const { stringify, URL } = require('url')`),
though I'm not seeing it elsewhere so assumed it was taboo in koajs/koa.